contracts/implementation/SMAOracle.sol

```diff
@@ -191,6 +198,16 @@ contract SMAOracle is IOracleWrapper {
         return sum / int256(k);
     }
 
+    /**
+     * @notice Changes the address of the associated `PoolKeeper` contract
+     * @param _poolKeeper Address of the new contract
+     * @dev Only callable by the deployer of this SMA oracle
+     */
+    function setPoolKeeper(address _poolKeeper) public onlyDeployer {
```

Comment:
I think we agreed that this would be pool governance. Just because this greatly increases our trust assumptions (adds a degree of mutability to any SMAOracle market)

Reply:
As in, `onlyDeployer` -> `onlyGov`?

Reply:
Yep